packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/backups.html

```diff
@@ -12,20 +12,25 @@ <h2 data-translate="pci_databases_backup"></h2>
     <oui-tile
         heading="{{:: 'pci_databases_backup_informations_title' | translate}}"
     >
-        <div class="oui-table">
-            <div>
-                <div class="oui-table__row">
-                    <div class="oui-table__cell">
-                        <div class="oui-description">
-                            <div
-                                data-translate="pci_databases_backup_informations_retention"
-                            ></div>
-                            <div>
-                                {{:: $ctrl.backupRetentionTime.humanize()}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
+        <div class="row px-3">
+            <div class="col border-top border-bottom border-light p-2">
```

Comment:
Hi @antleblanc, the padding here is used to have the same look as in the mockups :

**Mockup:** 
![image](https://user-images.githubusercontent.com/24541481/177713809-0c488fb9-1e63-4e47-8b49-59ab3218aa49.png)

**With padding:**
![image](https://user-images.githubusercontent.com/24541481/177712941-68ef439d-d089-49d1-bde9-829e7dff1921.png)

**Without padding:**
![image](https://user-images.githubusercontent.com/24541481/177712979-2f2d4bf5-a7ce-4e5f-817d-883991eb610c.png)

I could have used a `<table class="oui-table">`, but it would imply to add a custom css to add a `vertical-align: top;` rule: 
![image](https://user-images.githubusercontent.com/24541481/177715709-01a43829-2aea-4922-ad99-83ae66560719.png)

I also noticed that the `table` was overflowing the `oui-tile`  on lower resolutions:
![image](https://user-images.githubusercontent.com/24541481/177716145-54f721ea-adca-4a67-a249-6a3b2603a6a6.png)